.gitattributes

```diff
@@ -1,2 +1,3 @@
 *.wav filter=lfs diff=lfs merge=lfs -text
 *.mp3 filter=lfs diff=lfs merge=lfs -text
+assets/sounds/general/bingBong.mp3 filter=lfs diff=lfs merge=lfs -text
```

Comment:
Is this added separately from the rest of the mp3 files? Why does line 2 not include this?

Reply:
Good point. I'll change that ; )